calendar_reboot.js

```diff
@@ -129,6 +135,21 @@ $(document).ready(function() {
         return true;
     };
     
+    var validateForm = function(startDateString, endDateString) {
+        
+        
+        var validateTitle = validateInput(calendarTitleId);
+        var validateStartDate = validateInput(startDateId);
+        var validateEndDate = validateInput(endDateId);
+        
+        var isValid = validateTitle && validateStartDate && validateEndDate;
+        
+        return (isValid && validateDates(startDateString, endDateString);
```

Comment:
Just a note that this only works because `&&` short-circuits and so `validateDates` only gets called when `isValid` is true.